src/status_im2/subs/communities.cljs

```diff
@@ -329,3 +329,10 @@
  :<- [:communities]
  (fn [communities [_ id]]
    (get-in communities [id :images])))
+
+(re-frame/reg-sub
+ :communities/rules
+ :<- [:communities]
+ (fn [communities [_ community-id]]
+   (get-in communities [community-id :introMessage])))
```

Comment:
maybe worth renaming this in `src/status_im/communities/core.cljs`? instead of `introMessage`, it should be `intro-message`

Reply:
tks! Just pushed !